ci-operator/step-registry/openshift/e2e/aws/calico/openshift-e2e-aws-calico-workflow.yaml

```diff
@@ -0,0 +1,26 @@
+workflow:
+  as: openshift-e2e-aws-calico
+  steps:
+    pre:
+    - chain: ipi-conf-aws
+    - ref: network-conf-disable-cno
+    - ref: network-calico
+    - chain: ipi-install
+    test:
+    - ref: openshift-e2e-test
+    env:
+      # 1) skip maxUnavailable test as openshift-kube-proxy doesn't pass it
+      # 2) skip router idling test as this is OVN/openshiftSDN feature only
```

Comment:
can we get bug links for these, and/or improved skipping in origin?

Reply:
Removed this skip, its now in master after https://bugzilla.redhat.com/show_bug.cgi?id=1957146 was fixed